electron/js/menu/system.js

```diff
@@ -298,6 +311,21 @@ if (process.platform === 'win32') {
   });
 }
 
+if (process.platform === 'linux') {
+  menuTemplate.unshift(linuxTemplate);
+  editTemplate.submenu.push(separatorTemplate, {
+    i18n: 'menuPreferences',
+    click: function() {sendAction('profile-settings-show');},
+  });
+  windowTemplate.submenu.push(
+    separatorTemplate,
+    toggleMenuTemplate,
```

Comment:
remove the menu toggle from here.. doesn't seem to do much, there will be changes in Windows soon here so we'll rethink about it in the future